lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -425,25 +425,27 @@ sub run {
     my %args = @_;
 
     my $result = {};
-    my $ua     = Mojo::UserAgent->new;
-    $ua->connect_timeout(15);
 
-    my $config = config(%args);
+    my $config  = config(%args);
+    my $retries = $args{retries} // 3;
 
     foreach my $id (sort keys %$config) {
+        my $retry_counter = 0;
```

Comment:
where do we use this variable?

Reply:
It was added earlier, forgot to remove, thanks for mention it